### ISSUE TITLE
show_value breaks the rangeselector

### ISSUE THREAD

**Issue by rwegnerGermany** [dbuezas/lovelace-plotly-graph-card#196]
**Describe the bug**
With active show_value, the x axis range appears to be calculated wrong when pressing a rangeselector button.
Timezone is GMT+1.
Tested Firefox and Edge.

**Screenshots**
The initial view:
![default range](https://user-images.githubusercontent.com/69688683/213876496-83c5ab71-dd9c-4a80-b3cb-5c4f691c17eb.png)

After pressing the 2h button with **show_value: true**
![2h show_value true](https://user-images.githubusercontent.com/69688683/213876693-49eb2f66-d005-4841-879c-f786e7a7164b.png)

After pressing the 2h button with **show_value: false**
![2h show_value false](https://user-images.githubusercontent.com/69688683/213876697-4de4fff7-ed9d-4d8f-ac63-5a84a8a2f1b3.png)
ressing the 2h button with show_value: false

**yaml**
```yaml
type: custom:plotly-graph
entities:
  - entity: sensor.grid_power_10min_avg
    show_value:
      right_margin: 15
    line:
      color: rgb(255,127,14)
      width: 1.5
hours_to_show: 16
refresh_interval: 30
layout:
  xaxis:
    rangeselector:
      x: -0.1
      'y': 1.2
      buttons:
        - count: 1
          step: all
        - count: 2
          step: hour
          stepmode: backward
```

**Additional context**

**Comment by dbuezas** [comment]
Uhm. I hadn't thought of that.
In any case, the next release uses a different approach to show_value that doesn't fiddle with the ranges (which should fix this)

**Comment by dbuezas** [comment]
Closed by the release of v3.0.0
